.github/workflows/test_build_deploy.yml

```diff
@@ -74,15 +74,17 @@ jobs:
         uses: c-hive/gha-yarn-cache@v1
 
       - name: Authenticate git clone
+        if: env.DISTRIBUTION == 'wire' || github.event_name != 'pull_request'
         env:
           GH_TOKEN: ${{secrets.OTTO_THE_BOT_GH_TOKEN}}
-        run: echo -e "machine github.com\n  login ${GH_TOKEN}" > ~/.netrc
+        run: echo "machine github.com login ${GH_TOKEN}" > ~/.netrc
```

Comment:
Next line is not needed, see https://www.gnu.org/software/inetutils/manual/html_node/The-_002enetrc-file.html